src/py42/clients/detectionlists/high_risk_employee.py

```diff
@@ -127,4 +133,27 @@ def get_all(self, filter_type=u"OPEN", sort_key=None, sort_direction=None):
             filter_type=filter_type,
             sort_key=sort_key,
             sort_direction=sort_direction,
+            page_size=100,
         )
+
+    def _get_high_risk_employees_page(
+        self,
+        tenant_id,
+        filter_type=None,
+        sort_key=None,
+        sort_direction=None,
+        page_num=None,
+        page_size=None,
+    ):
+        # Overwriting page_size since default value 1000 returns error
```

Comment:
This comment should be moved to `get_all`, as that is where we set value to 100.